polaris/ocean/config/single_layer.yaml

```diff
@@ -0,0 +1,19 @@
+omega:
+  time_integration:
+    config_time_integrator: RK4
```

Comment:
I have run some single layer cases with split-explicit, suggesting that it isn't the case that it only works with RK4. Are there known issues with single layer split-explicit or have we just been assuming that it doesn't work? Or is it the case that we just need someone to more fully test split-explicit in single layer cases?